cnd/src/http_api/routes/index.rs

```diff
@@ -163,6 +196,29 @@ impl ToCreatedSwap<herc20::CreatedSwap, halight::CreatedSwap>
     }
 }
 
+impl ToCreatedSwap<halight::CreatedSwap, herc20::CreatedSwap>
```

Comment:
Isn't this trait supposed to be gone?

Reply:
The trait is back! There is a generic blanket implementation for it in this PR: https://github.com/comit-network/comit-rs/pull/2719

Reply:
Yep, good catch.

Reply:
> The trait is back! There is a generic blanket implementation

Not quite. It's a generic blanket implementation for the struct `Body<A,B>`, but there is no trait.

Reply:
Yep, minor difference @tcharding :)

Strictly speaking, what I did in #2719 is not a blanket impl because it doesn't use a trait. All what I did was defining a generic function where certain trait bounds have to be fulfilled for it to be called. Less magic than a blanket impl actually.